library-api/src/main/java/cz/muni/fi/pa165/library/facade/SingleLoanFacade.java

```diff
@@ -0,0 +1,36 @@
+package cz.muni.fi.pa165.library.facade;
+
+import cz.muni.fi.pa165.library.dto.BookDTO;
+import cz.muni.fi.pa165.library.dto.SingleLoanDTO;
+import cz.muni.fi.pa165.library.dto.SingleLoanDTO.UserDTO;
+
+import java.util.List;
+
+/** @author Martin Páleník 359817 */
+public interface SingleLoanFacade {
+
+    /** The time of loan will be set to now() */
+    public void borrowBook(BookDTO bookDto, UserDTO userDto);
+
```

Comment:
Use SingleLoanDto instead of those two dtos

Reply:
Also, I think there should be a method create Loan in LoanFacade, which creates multiple single loans . So in this class you would provide createSingleLoan instead